HintList.py

```diff
@@ -198,79 +221,763 @@ def getRequiredHints(world):
+    'Ganons Castle Light Trial Lullaby Chest':                     ("#music in the test of radiance# reveals", None, 'exclude'),
+
+    'Ganons Castle MQ Water Trial Chest':                          ("the #test of the seas# holds", None, 'exclude'),
+    'Ganons Castle MQ Forest Trial Eye Switch Chest':              ("the #test of the wilds# holds", None, 'exclude'),
+    'Ganons Castle MQ Forest Trial Frozen Eye Switch Chest':       ("the #test of the wilds# holds", None, 'exclude'),
+    'Ganons Castle MQ Light Trial Lullaby Chest':                  ("#music in the test of radiance# reveals", None, 'exclude'),
+    'Ganons Castle MQ Shadow Trial Bomb Flower Chest':             ("the #test of darkness# holds", None, 'exclude'),
+    'Ganons Castle MQ Shadow Trial Eye Switch Chest':              ("the #test of darkness# holds", None, 'exclude'),
+    'Ganons Castle MQ Spirit Trial Golden Gauntlets Chest':        ("#reflected light in the test of the sands# reveals", None, 'exclude'),
+    'Ganons Castle MQ Spirit Trial Sun Back Right Chest':          ("#reflected light in the test of the sands# reveals", None, 'exclude'),
+    'Ganons Castle MQ Spirit Trial Sun Back Left Chest':           ("#reflected light in the test of the sands# reveals", None, 'exclude'),
+    'Ganons Castle MQ Spirit Trial Sun Front Left Chest':          ("#reflected light in the test of the sands# reveals", None, 'exclude'),
+    'Ganons Castle MQ Spirit Trial First Chest':                   ("#reflected light in the test of the sands# reveals", None, 'exclude'),
+    'Ganons Castle MQ Spirit Trial Invisible Chest':               ("#reflected light in the test of the sands# reveals", None, 'exclude'),
+    'Ganons Castle MQ Forest Trial Freestanding Key':              ("the #test of the wilds# holds", None, 'exclude'),
+
+    'Deku Tree Queen Gohma Heart':                                 ("the #Parasitic Armored Arachnid# guards", "#Queen Gohma# guards", 'exclude'),
+    'Dodongos Cavern King Dodongo Heart':                          ("the #Infernal Dinosaur# guards", "#King Dodongo# guards", 'exclude'),
+    'Jabu Jabus Belly Barinade Heart':                             ("the #Bio-Electric Anemone# guards", "#Barinade# guards", 'exclude'),
+    'Forest Temple Phantom Ganon Heart':                           ("the #Evil Spirit from Beyond# guards", "#Phantom Ganon# guards", 'exclude'),
+    'Fire Temple Volvagia Heart':                                  ("the #Subterranean Lava Dragon# guards", "#Volvagia# guards", 'exclude'),
+    'Water Temple Morpha Heart':                                   ("the #Giant Aquatic Amoeba# guards", "#Morpha# guards", 'exclude'),
+    'Spirit Temple Twinrova Heart':                                ("the #Sorceress Sisters# hold", "#Twinrova# guards", 'exclude'),
+    'Shadow Temple Bongo Bongo Heart':                             ("the #Phantom Shadow Beast# guards", "#Bongo Bongo# guards", 'exclude'),
```

Comment:
Should these all be hold instead of guard?

Reply:
Changed to "holds" for all. I can go either way on these.